### ISSUE TITLE
Custom icon classnames are gone again

### ISSUE THREAD

**Issue by jobee** [neos/neos-ui#1994]
### Description

Missing icons in backend

### Steps to Reproduce

1. Change a document icon via NodeTypes.yaml e.g. `icon: 'fas fa-tag super-important'`
2. Refresh the nodetree

#### Expected behavior

Nodetree icon in front of the document label incl. the custom class

#### Actual behavior

No icon is shown for the solid iconset (fas).
However the brand iconset (fab) still works, eg. `icon: 'fab fa-product-hunt super-important'`

### Affected Versions

Neos: 3.3.12

UI: >=1.1.2

**Comment by mstruebing** [comment]
Hey sorry to hear that, I'm sure someone will work on this soonish  to fix this.

**Comment by h1Modeling** [comment]
Hi @mstruebing, I've looked into this and also found the reason why it doesn't work. 
I'm just wondering how the functionality is supposed to be:
1. Should icons in general be provided without the prefixes? Or should both be possible?
2. Should all FA5 icons be inside the mapping file or is there a reason why there is a reduced amount in that file (on the FA webpage it lists over 1000 icons and in the mapping file there are just around 500)

I will provide a PR after I got the information. Thanks in advance.
